README.md

```diff
@@ -220,6 +221,45 @@ C:\Users\bion\Desktop>dumpbin /IMPORTS .\example.exe | findstr msvcp
     msvcp140d_oss.dll
 ```
 
+# How To Run The Tests
+
+In order to run any of the tests you must be in a correctly configured environment for the architecture you are
+targeting, have python 3.8 or newer and have `clang-cl` on the PATH. If you are testing the x64 flavor of the STL then
+running from the "x64 Native Tools Command Prompt" is sufficient for setting up the environment.
+
+## Running All The Tests
+
+After configuring the project, running `ctest` from the root of the build output directory will run all the tests.
+CTest will only display the standard error output of tests which failed. In order to get more details from CTest's

```

Comment:
"which" is nonrestrictive, "that" is restrictive, so this should be:

```suggestion
CTest will only display the standard error output of tests that failed. In order to get more details from CTest's
```

Reply:
This doesn't look resolved to me.

Reply:
Thanks for catching that. I usually click resolved as I fix suggestions. Misaligned on this one I guess.